src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -91,6 +98,10 @@ export class EssentialsComponent extends BaseControllerDirective {
       this.nodes.refresh(messageHandler).pipe(map(() => {this.updateItemInEssentials(); })),
       this.systemApp.refresh(messageHandler).pipe(catchError(err => of(null))),
       this.clusterUpgradeProgress.refresh(messageHandler),
+      this.RestClient.getPartitionById(this.data.getFailoverManagerPartition(), messageHandler).pipe(map((partition) => {
```

Comment:
given the FM partition ID is a constant, we should set it as a constant.

Reply:
Should I create a new export in Common/Constants.ts that includes several service partition Ids?

Reply:
Yes that works